src/main/java/com/ibm/cloudant/kafka/connect/CloudantSourceTask.java

```diff
@@ -201,20 +200,7 @@ public void start(Map<String, String> props) {
                 }
             }
 
-            // Create a new CloudantClient instance for account endpoint account.cloudant.com
-            String urlWithoutProtocal = url.substring(url.indexOf("://") + 3);
-            String account = urlWithoutProtocal.substring(0, urlWithoutProtocal.indexOf("."));
-
-            CloudantClient cantClient = ClientBuilder.account(account)
-                    .username(userName)
-                    .password(password)
-                    .build();
-
-            // Create a database instance
-            String cantDBName = url.substring(url.lastIndexOf("/") + 1);
-
-            // Create a database instance
-            cantDB = cantClient.database(cantDBName, false);
+            cantDB = JavaCloudantUtil.getDBInst(url, userName, password, false);
```

Comment:
The variable name `cantDB` should be updated to something more descriptive

Reply:
Fixed in a35d0bd0e3ef803441c52b25163f1903b9deab90